documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -37,7 +37,13 @@ spec:
 |Label |Description |Default value
 
 |`controller_max_vm_inflight`
-|The maximum number of VMs per plan that can be migrated simultaneously.
+a|*Varies with provider as follows*:
+
+* {rhv-full}, {osp}, VMware warm migrations _or_ remote cold migrations: The maximum number of disks that {project-short} can transfer simultaneously.
+
+* OVA or VMware local cold migrations: The maximum number of VMs that {project-short} can migrate simultaneously.
```

Comment:
For **all** occurrences of   **local cold migrations**:
I suggest to explain what does "local migration" mean and therefore rephrase to something like:
 **cold migrations to a local OpenShift cluster**

Reply:
It does not apply for **all** local cold migrations but afaik only for virt-v2v related migrations.